src/cascadia/TerminalConnection/ConhostConnection.h

```diff
@@ -26,20 +26,22 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         winrt::event<TerminalConnection::TerminalOutputEventArgs> _outputHandlers;
         winrt::event<TerminalConnection::TerminalDisconnectedEventArgs> _disconnectHandlers;
 
-        uint32_t _initialRows{};
-        uint32_t _initialCols{};
-        hstring _commandline{};
-        hstring _startingDirectory{};
-
-        bool _connected{};
-        HANDLE _inPipe{ INVALID_HANDLE_VALUE };  // The pipe for writing input to
-        HANDLE _outPipe{ INVALID_HANDLE_VALUE }; // The pipe for reading output from
-        HANDLE _signalPipe{ INVALID_HANDLE_VALUE };
-        DWORD _outputThreadId{};
-        HANDLE _hOutputThread{ INVALID_HANDLE_VALUE };
-        PROCESS_INFORMATION _piConhost{};
-        guid _guid{}; // A "unique" session identifier for connected client
-        bool _closing{};
+        // set during construction - not constant
+        uint32_t _initialRows;
+        uint32_t _initialCols;
+        hstring _commandline;
+        hstring _startingDirectory;
+        guid _guid; // A "unique" session identifier for connected client
+
+        bool _connected{ false };

```

Comment:
I realize that `{}` is equivalent to `{false}`, but it seems important to me to say `{false}` or `{true}` because a boolean has only two states.

Reply:
`{}` implies default value initialization, so yes, it is the same.  Personally, the only reason I prefer {} over {false}, is because when there a large number of class members declared together, {true} sticks out very obviously:

```C++
bool member1{};
bool member1{};
bool member2{};
bool member3{true};
bool member4{};
```

However, it's a matter of personal preference, so whatever the project prefers is fine.

Reply:
I personally prefer `{false}` because it subtracts the mental load of evaluating the default for a type in my head.

Reply:
I sort of understand, but the default for every type though is effectively zero.  That doesn’t seem very mysterious to me?

Reply:
I chose `{}` except for the atomic, which requires initialization.